setup.py

```diff
@@ -107,7 +107,10 @@ def get_git_sha():
         'lxml==3.8.0',
         'numexpr==2.6.4',
         'xlrd==1.1.0',
-        'python-ldap==3.1.0'
+        'python-ldap==3.1.0',
+        'requests-kerberos==0.12.0',
+        'django==2.1.5'
```

Comment:
are we using anything specific from django? django and FAB are not complementary framework

Reply:
ok. just as note wehen we raise open source PR we **may** need to move to other equivalent utils

Reply:
sure